### ISSUE TITLE
POAP rewards can be claimed from completed Quests

### ISSUE THREAD

**Issue by olavra** [decentraland/unity-renderer#185]
As soon as the players completes a Quests with associated POAP rewards, they will be able to claim them by going to the Quest Log 

![image.png](https://images.zenhubusercontent.com/603cdb58edb53386a3d0d8d5/5c3b813a-6a31-49db-b1d7-4949b1c339e6)

or directly in the Quest Completed notification:

![image.png](https://images.zenhubusercontent.com/603cdb58edb53386a3d0d8d5/890b557d-d0e5-4b11-948a-7d306e4e5ba4)

**Comment by AjimenezDCL** [comment]
I dont think this is needed anymore. POAPs are collected automatically without user interaction.
